integration-core/pom.xml

```diff
@@ -17,5 +17,13 @@
             <artifactId>integration-healthcheck</artifactId>
             <version>${project.version}</version>
         </dependency>
+
+        <dependency>
+            <groupId>org.symphonyoss</groupId>
+            <artifactId>integration-webhook</artifactId>
+            <version>${project.version}</version>
+            <type>test-jar</type>
+            <scope>test</scope>
+        </dependency>
```

Comment:
It has no circular dependency because the integration-webhook module doesn't know about the integration-core module, but I agree with you.

I'll move the common test classes to integration-commons module to reuse in other modules.